libs/sdk-ui-kit/styles/scss/variables.scss

```diff
@@ -25,7 +25,11 @@ $gd-color-dark: #000;
 $border-color: #dde4eb;
 
 // secondary colors
-$gd-color-highlight-dimmed: mix($gd-color-highlight, $gd-color-text-light, 10%);
+// $gd-color-highlight-dimmed: mix($gd-color-highlight, $gd-color-text-light, 10%);
```

Comment:
This commented-out code should probably be removed.

Reply:
fixed